cli/network.ts

```diff
@@ -269,15 +288,17 @@ export const deployContractAndSave = async (
         ? deployResult.libraries
         : undefined,
   })
+  logger.log('> Contract saved to address book')
 
   return deployResult.contract
 }
 
 export const deployContractWithProxyAndSave = async (
   name: string,
-  args: Array<{ name: string; value: string }>,
+  args: Array<any>,
```

Comment:
The issue with this being any is that if we pass an array with no parameter names then the entry will not be saved with the same format. Let me think about a refactor